### PR TITLE
20240630 hex content

### PR DESCRIPTION
**Pull request by nxSerhey**

### CODE REVIEW

File: src/control/HdrHomeButton/template.mjs [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const CLASS = representClassNames({
 export const HTML = `
 <a class="${CLASS.ROOT} notranslate" translate="no" href="\${ENV:HOST_URL}" draggable="false">
   <div></div>
-  <span>Home</span>
+  <span>Huem</span>
```
Comment (yacubin) [inline src/control/HdrHomeButton/template.mjs]: Why is renamed to Huem

File: src/control/HexContent/template.mjs [hunk 2]
```diff
@@ -184,21 +205,21 @@ ${DARKMODE_SELECTOR_VALUE}
 
 .${CLASS.BINARY}
 {
-  border-right: 1px solid var(--whex-cont-bor);
-  border-left: 1px solid var(--whex-cont-bor);
+  /*border-right: 1px solid var(--whex-cont-bor);*/
```
Comment (yacubin) [inline src/control/HexContent/template.mjs]: Why are these comments left?

File: src/control/HexContent/template.mjs [hunk 3]
```diff
@@ -184,21 +205,21 @@ ${DARKMODE_SELECTOR_VALUE}
...
+  /*border-left: 1px solid var(--whex-cont-bor);*/
```
Comment (yacubin) [inline src/control/HexContent/template.mjs]: Why are these comments left?

File: src/control/HexContent/template.mjs [hunk 4]
```diff
@@ -184,21 +205,21 @@ ${DARKMODE_SELECTOR_VALUE}
...
 .${CLASS.BINARY} > div
 {
-  padding: 20px 10px 10px 10px;
+  padding: 10px 10px 10px 10px;
 }
 
 .${CLASS.CONTENT} h3
 {
-  background-color: var(--whex-title-bg);
+  /*background-color: var(--whex-title-bg);*/
```
Comment (yacubin) [inline src/control/HexContent/template.mjs]: Why are these comments left? Should also be removed from root and darkmode variable blocks.

File: src/control/HdrHomeButton/template.mjs (hunk 1 above)
Comment (nxSerhey) [reply src/control/HdrHomeButton/template.mjs]: Fix

File: src/control/HexContent/template.mjs (hunk 4 above)
Comment (nxSerhey) [reply src/control/HexContent/template.mjs]: deleted

File: src/control/HexContent/template.mjs (hunk 3 above)
Comment (nxSerhey) [reply src/control/HexContent/template.mjs]: deleted

File: src/control/HexContent/template.mjs (hunk 2 above)
Comment (nxSerhey) [reply src/control/HexContent/template.mjs]: deleted

**Review by yacubin — APPROVED** [review]

**Pull request merged**